packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/angular/load-app.md

```diff
@@ -0,0 +1,80 @@
+The first step for any application is to install or embed the Widget. You can link out to the Okta CDN or install locally through a package manager (for example, `npm`).
+
+### To use the CDN, include the following tags in your HTML:
+
+```html
+<!-- Latest CDN production JavaScript and CSS -->
+<script src="https://global.oktacdn.com/okta-signin-widget/-=OKTA_REPLACE_WITH_WIDGET_VERSION=-/js/okta-sign-in.min.js" type="text/javascript"></script>
+<link href="https://global.oktacdn.com/okta-signin-widget/-=OKTA_REPLACE_WITH_WIDGET_VERSION=-/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
+```
+
+More information, including the latest published version, is available in the [Okta Sign-In Widget Documentation](https://github.com/okta/okta-signin-widget#using-the-okta-cdn).
```

Comment:
```suggestion
More information, including the latest published version, is available in the [Okta Sign-In Widget documentation](https://github.com/okta/okta-signin-widget#using-the-okta-cdn).
```

Reply:
I changed this to `Okta Sign-In Widget Documentation`, which is the official title of the repo